tms_objects2.erb

```diff
@@ -490,9 +490,18 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
                   <lido:conceptID lido:source="<%= PCAATCN %>" lido:type="Publication related role"><%= PCAATID %></lido:conceptID>
                   <lido:term><%= pc["Role"] %></lido:term>
                 </lido:roleActor>
+        <% PubConLifeRole.each do |lr| %>
```

Comment:
example object where  there's one or more PubConLifeRole

Reply:
Although not in the coboat like this, it would be better to have the enclosing roleActor only within the if for
lr["conID"] == pc["ConstituentID"]

Reply:
Still need an object that has PubConLifeRole to check this.  (615 doesn't have this AFAICT)

<% PubConLifeRole.each do |lr| %>
        <% if lr["conID"] == pc["ConstituentID"] %>
                <lido:roleActor>
                  <lido:conceptID lido:source="<%= lr["UCTS"] %>" lido:type="<%= lr["thesxreftype"] %>"><%= lr["FinalST"] %></lido:conceptID>
                  <lido:term><%= lr["Term"] %></lido:term>
                </lido:roleActor>
        <% end %>
        <% end %>   <%# PubConLifeRole |lr| %>